jest/private/jest_test.bzl

```diff
@@ -58,7 +57,6 @@ _attrs = dicts.add(js_binary_lib.attrs, {
 def _impl(ctx):
     providers = []
     generated_config = ctx.actions.declare_file("%s__jest.config.mjs" % ctx.label.name)
-    user_config = copy_file_to_bin_action(ctx, ctx.file.config) if ctx.attr.config else None
```

Comment:
Why don't we need to copy this to the output tree anymore?

Reply:
In the case that it's a file rather than a js_library.

Reply:
Looks like I can revert this part and continue to use the copied one, I'm not sure why though? Is there a test we should add to show why it's required?

The original reason I did it is because the attribute must be used down below to access the providers and the target itself.

Reply:
My understanding is that we put everything that can resolve files in the output tree to prevent escaping out of the output tree.

Reply:
The rationale is [here](https://github.com/aspect-build/rules_js#running-nodejs-programs).